expyfun/visual/_visual.py

```diff
@@ -826,6 +826,76 @@ def __init__(self, ec, colors=('w', 'k')):
         self.set_radius(1, 1, units='pix')
 
 
+class ProgressBar(object):
+    """A progress bar that can be displayed between sections
+
+    This uses two rectangles, one outline, and one solid to show how much
+    progress has been made in the experiment.
+
+    Parameters
+    ----------
+    ec : instance of ExperimentController
+        Parent EC.
+    pos : array-like
+        4-element array-like with X, Y center and width, height.
+    units : str
+        Units to use. These will apply to all spatial aspects of the drawing.
+        Must be either ''norm'' or ''pix''.
+    colors : list or tuple of matplotlib Colors
+        Colors to fill and outline the bar respectively. Defaults to green and
+        white.
+    text : bool
+        Whether to show the percent done as text below the bar. If ''True'',
```

Comment:
shouldn't this be double-backticks, not doubled single quotes?

Reply:
Thanks! And so sorry for my many confusions about convention with the backticks!